Car_Localizer/Car_Localizer/Box/box.py

```diff
@@ -0,0 +1,5 @@
+import cv2
+
+
+def draw_box(image, x, y, width, height):
```

Comment:
Lets use typing module and check if it works with py torch, add a bounding_box_tests.py and plot image with that box